unison-src/transcripts-using-base/binary-encoding-nats.output.md

```diff
@@ -77,18 +76,10 @@ testABunchOfNats _ =
 ``` ucm
 scratch/main> add
 
-  ⍟ I've added these definitions:
-
-    type EncDec
-    BE16             : EncDec
-    BE32             : EncDec
-    BE64             : EncDec
-    LE16             : EncDec
-    LE32             : EncDec
-    LE64             : EncDec
-    testABunchOfNats : ∀ _. _ ->{IO} [Result]
-    testNat          : Nat -> '{IO, Stream Result} ()
-    testRoundTrip    : Nat -> EncDec ->{IO, Stream Result} ()
```

Comment:
I already somewhat miss this, but maybe the slurp result v2 will be enough, so we can try that first before worrying about replacing this functionality.

Reply:
I agree it would be nicer to say something other than `Done` in the `update` feedback, but you do pretty much get this exact `add` output _before_ running `update`, so I guess that's part of the reason it hasn't been on our radar to make it more verbose.

But it'd be trivial to add back, plus more – you might like to see number of propagated updates, for example

Reply:
Yeah agreed about the slurp output. I was imagining we could eventually put the number of propagated updates there too or instead.